script.js

```diff
@@ -17,3 +17,121 @@ navlinks.addEventListener('click', () => {
+    technologies: ['html', 'css', 'javascript'],
+    source_link: 'https://github.com/tahseenzahra/microverse-template-portfolio',
+    version_link: 'https://tahseenzahra.github.io/microverse-template-portfolio/',
+  },
+  {
+    id: 'project4',
+    title: 'Tonic',
+    info: ['CANOPY', 'Back End Dev', '2022'],
+    featured_img: './images/img4.svg',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    more_description: 'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Laborum, nobis!',
+    technologies: ['html', 'css', 'javascript'],
+    source_link: 'https://github.com/tahseenzahra/microverse-template-portfolio',
+    version_link: 'https://tahseenzahra.github.io/microverse-template-portfolio/',
+  },
+];
+// Project detail popup Data *end
+
+const arrindex = [];
+
+const prjtitle = document.querySelector('#proj-title-id');
+
+// const prjinfo = document.querySelector('#proj-about-id');
+const prjname = document.querySelector('#proj-name-id');
+const prjtype = document.querySelector('#proj-type-id');
+const prjyear = document.querySelector('#proj-year-id');
+const prjimg = document.querySelector('#img-id');
+const prjdescription = document.querySelector('#proj-primarytxt-id');
+const prjtechnologies = document.querySelector('#tech-list-id');
+const prjSrcbtn = document.querySelector('#sourcebtn-id');
+const prjLivebtn = document.querySelector('#livebtn-id');
+
+// Function GetData for detail project popup * start
+
+function getData(i) {
+  prjtitle.textContent = projectsDetail[i].title;
+
+  const [projectName, projectRole, projectYear] = projectsDetail[i].info;
+  prjname.textContent = projectName;
+  prjtype.textContent = projectRole;
+  prjyear.textContent = projectYear;
+
+  prjimg.src = projectsDetail[i].featured_img;
+
+  prjdescription.textContent = projectsDetail[i].more_description;
+
+  let j = 0;
+  do {
+    const technology = document.createElement('li');
+    technology.textContent = projectsDetail[arrindex].technologies[j];
+
+    // technology.className = 'popup-tech-list-li';
+    prjtechnologies.appendChild(technology);
+    j += 1;
+  } while (j < projectsDetail[i].technologies.length);
+
+  prjSrcbtn.href = projectsDetail[i].source_link;
+  prjLivebtn.href = projectsDetail[i].version_link;
+}
+
+// function to get index i value from see project btn id
+
+function found(input) {
+  for (let i = 0; i < projectsDetail.length; i += 1) {
+    if (projectsDetail[i].id === input) {
+      arrindex.push(i);
+      getData(i);
+
+      return i;
+    }
+  }
+  return null;
+}
+
+// function to get string to match id value from see Project BTN
+const detailPopup = document.getElementById('popup-1');
+function popup(string) {
+  detailPopup.style.display = 'block';
+  found(string);
+}
+popup();
+
+const popupclosebtn = document.querySelector('#x-icon-popup-id');
+popupclosebtn.addEventListener('click', () => {
+  detailPopup.style.display = 'none';
```

Comment:
While you did a good job implementing the pop-up window. I think it would be nice to stick to the project design. The pop up is supposed to be displayed as a whole without scrolling up or down. Please make some adjustments. 

This is your design: 
![Screenshot from 2022-07-23 13-24-12](https://user-images.githubusercontent.com/79506105/180599606-21b0b6dd-0eb4-4253-8c77-c15d379fbce3.png)

Please stick to this FIGMA design:
![Screenshot from 2022-07-23 13-34-13](https://user-images.githubusercontent.com/79506105/180599720-5eb60a46-a810-4546-b123-4ad7e262769f.png)